packages/mui/README.md

```diff
@@ -72,13 +72,12 @@ This will populate the /icons directory with all of the icon components as well
 To publish the package:
 
 ```
-yarn build:full
-cd icons
+yarn build
 npm version < patch | minor | major >
-npm publish
+npm publish:package
```

Comment:
```suggestion
npm publish
```

publish:package is an npm script that we created. `npm publish` is an NPM CLI command.
It either needs to be `yarn publish:package` or `npm publish`